docs/upgrading/splinter-v0.3.15-from-v0.3.14.md

```diff
@@ -0,0 +1,8 @@
+# Upgrading to Splinter v0.3.15 from Splinter v0.3.14
+
+Breaking changes and significant changes in this release
+
+* The routes for `biome/users/{user_id}/keys` and
+  `biome/users/{user_id}/keys/{public_key}` have changed to `biome/keys` and
+  `biome/keys/{public_key}` respectively. The user id will be derived from
+  the provided access token.
```

Comment:
As this provided access token is new, we should also add information about this to the migration guide.

Reply:
I can add something about the access token, but the access token isn't new for 0.3.15. It was always required by the keys routes, it's just that the routes weren't extracting the user id from it. So, from the perspective of an API user the only change is the path.

Reply:
Doesn't the user also need to ensure that they are providing a header?  If so, we haven't documented that part yet.

Reply:
Are talking about in general or in the upgrade doc?